src/System.Reflection.Emit.ILGeneration/tests/CustomAttributeBuilder/CustomAttributeBuilderCtor2.cs

```diff
@@ -0,0 +1,1010 @@
+        {
+            this.TestStringField = testString;
+            this.TestInt = testInt32;
+            this.GetString = getOnlyString;
+            this.GetInt = getOnlyInt32;
+        }
+    }
+
+    public class CustomAttributeBuilderCtor2
+    {
+        private const int c_MIN_STRING_LENGTH = 1;
+        private const int c_MAX_STRING_LENGTH = 1024;
+        private const string c_FIELD_TESTINT32_NAME = "TestInt";
+        private const string c_FIELD_TESTSTRING_NAME = "TestStringField";
+        private const string c_FIELD_GETONLYSTRING_NAME = "GetString";
+        private const string c_FIELD_GETONLYINT_NAME = "GetInt";
+        private const string c_DEFAULT_NOT_EXIST_FIELDNAME = "DOESNOTEXIST";
+        private const BindingFlags c_FIELD_BINDING_FLAG = BindingFlags.Public | BindingFlags.Instance | BindingFlags.NonPublic;
+
+        [Fact]
+        public void PosTest1()
+        {
+            string str1 = null;
+            int testInt1 = 0;
+            int testInt2 = 0;
+
+            str1 = "PosTest1_Arg";
+            testInt1 = TestLibrary.Generator.GetInt32(-55);
+            testInt2 = TestLibrary.Generator.GetInt32(-55);
+
+            Type CustomAttributeBuilderTestType = typeof(CustomAttributeBuilderTest);
+
+            Type[] ctorParams = new Type[]
+            {
+                typeof(string),
+                typeof(int)
+            };
+            object[] constructorArgs = new object[]
+            {
+                str1,
+                testInt1
+            };
+            FieldInfo[] namedField = new FieldInfo[]
+            {
+                CustomAttributeBuilderTestType.GetField(c_FIELD_TESTINT32_NAME, c_FIELD_BINDING_FLAG)
+            };
+            object[] fieldValues = new object[]
+            {
+                testInt2
+            };
+
+            CustomAttributeBuilder cab = new CustomAttributeBuilder(
+                CustomAttributeBuilderTestType.GetConstructor(ctorParams),
+                constructorArgs,
+                namedField,
+                fieldValues);
+            Assert.NotNull(cab);
+
+            FieldInfo[] verifyFields = new FieldInfo[]
+            {
+                CustomAttributeBuilderTestType.GetField(c_FIELD_TESTINT32_NAME, c_FIELD_BINDING_FLAG),
+                CustomAttributeBuilderTestType.GetField(c_FIELD_TESTSTRING_NAME, c_FIELD_BINDING_FLAG),
+                CustomAttributeBuilderTestType.GetField(c_FIELD_GETONLYSTRING_NAME, c_FIELD_BINDING_FLAG),
+                CustomAttributeBuilderTestType.GetField(c_FIELD_GETONLYINT_NAME, c_FIELD_BINDING_FLAG),
+            };
+
+            object[] verifyFieldValues = new object[]
+            {
+                testInt2,
+                null,
+                str1,
+                testInt1
+            };
+
+            Assert.False(!VerifyCustomAttribute(cab, CustomAttributeBuilderTestType, verifyFields, verifyFieldValues));
```

Comment:
Can we get rid of the double negation? There's a couple of these here and in other files.